Package.swift

```diff
@@ -19,7 +19,7 @@ import PackageDescription
 
 let package = Package(
   name: "AppCheck",
-  platforms: [.iOS(.v11), .macCatalyst(.v13), .macOS(.v10_13), .tvOS(.v12), .watchOS(.v7)],
+  platforms: [.iOS(.v12), .macCatalyst(.v15), .macOS(.v10_15), .tvOS(.v13), .watchOS(.v7)],
```

Comment:
`.macCatalyst(.v15)` is breaking firebase-ios-sdk because firebase-ios-sdk's https://github.com/firebase/firebase-ios-sdk/actions/runs/9979855090/job/27579911476?pr=13301#step:6:93